README.rst

```diff
@@ -27,15 +27,29 @@ And an Array can be converted to a numpy ndarray on demand::
     array([ 0.72151309,  0.54654914], dtype=float32)
 
 
-Get in touch!
--------------
+------------------------------------------
+Important Note : Future development status
+------------------------------------------
+Biggus was originally developed primarily to provide large data handling for
+`Iris <http://github.com/SciTools/iris>`_ .
+
+For most applications it has now been surpassed by
```

Comment:
Using "most" here suggests there are still some activities better handled by biggus... and I'm not sure this is true beyond maybe existing code that still uses biggus.

Would "In general it has been surpassed by dask..." (or something!) be better?